split-llvm-extract.cpp

```diff
@@ -1,3 +1,10 @@
+/*
+ * A program that uses llvm-extract to split
+ * `*.bc` files into modules containing only
```

Comment:
It is an LLVM-split but I would say: Bitcode (`*.bc`) files.

Reply:
Yes, probably a good idea to say `LLVM bitcode (*.bc) files`.